src/System.Security.Cryptography.Cng/tests/Configurations.props

```diff
@@ -2,6 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
+      netfx-Windows_NT;
```

Comment:
Cng is no longer part of netstandard so why cannot it be netstandard? What errors do we get when this is netstandard?